src/analyses/apron/relationAnalysis.apron.ml

```diff
@@ -175,25 +175,32 @@ struct
       Since affine equalities can only keep track of integer bounds of expressions evaluating to definite constants, we now query the integer bounds information for expressions from other analysis.
       If an analysis returns bounds that are unequal to min and max of ikind , we can exclude the possibility that an overflow occurs and the abstract effect of the expression assignment can be used, i.e. we do not have to set the variable's value to top. *)
 
-  let no_overflow (ask: Queries.ask) exp =
+  let no_overflow_not_constraint (ask: Queries.ask) exp =
     match Cilfacade.get_ikind_exp exp with
-    | exception Invalid_argument _ -> false (* TODO: why this? *)
```

Comment:
Because I found out that get_ikind_exp throws an Invalid_argument exception when the expression is e.g. of type float. I could add a comment that explains this.

Reply:
It should be changed to a more meaningful comment, as we checked why this exception is caught here